public/views/listProblems/listProblems.html

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+    <!--
+    <div class="page-header">
```

Comment:
Done! I did leave the comments for displaying the language and topics, so we can do that easily later if we want to. But they aren't that big and would be easy to replace. Should I take them out too?